src/Dockerfile

```diff
@@ -59,8 +59,9 @@ ENV AIRFLOW_HOME=${AIRFLOW_USER_HOME}
 
 # Fix airflow bug with a patch. This fix comes from upstream Airflow v2.2.0
 # however, we cannot upgrade to that version atm.
-COPY patches/views.py.patch /tmp/views.py.patch
-RUN patch -d/ -p0 < /tmp/views.py.patch
+# We don't need this patch any more, since upgrade to v2.2.3
```

Comment:
Wat mij betreft haal je de patching helemaal weg. We kunnen het altijd nog terugvinden via git als het nodig is.